### PR TITLE
Add method on _query.dart to support regexp feature

### PR DESCRIPTION
**Pull request by sota1235**
Hello,  @isoos.
I'm one of the users of this library, it's really useful so I want to say thank you to you:)

On this PR, I added a code to support the features below.

- [Regexp query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-regexp-query.html) on ElasticSearch

I need it on my project, but there is no way to use this feature via this library.
So please confirm my PR and concern to merge it(or give me a comment to fix some points)

Thank you

### CODE REVIEW

**Comment by isoos** [comment]
@sota1235 this looks good, could you please also update the changelog and the pubspec.yaml (with a version bump)?

**Comment by sota1235** [comment]
sure, wait a moment plz... 🙏

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 0.3.6
+
+- Supporting [Regexp query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-regexp-query.html)
```
Comment (isoos) [inline CHANGELOG.md]: Could you please move this URL to the `_query.dart` file, and link only to the PR here? Make it similar to the previous ones...

Comment (sota1235) [reply CHANGELOG.md]: fixed 🔧

**Comment by isoos** [comment]
thanks, published as 0.3.6

**Comment by sota1235** [comment]
thank you so much!

**Pull request merged**